cluster/repository/cluster_repository.go

```diff
@@ -34,12 +34,12 @@ type Cluster struct {
 	LoggingURL string
 	// Application host name used by the cluster
 	AppDNS string
-	// Service Account token
-	SaToken string
+	// Encrypted Service Account token
```

Comment:
We don't know, it's encrypted or not by looking at this field. we need to look at `SATokenEncrypted`

Reply:
+1

Reply:
addressed in eb0aa0a